Components/Button/index.js

```diff
@@ -1,32 +1,20 @@
 import styled, { css } from "styled-components";
 
 const StyledButton = styled.button`
-  background-color: var(--primary-color);
+  background-color: ${(props) =>
+    props.isActive ? "var(--secondary-color)" : "var(--primary-color)"};
   border-radius: 12px;
   border: none;
   padding: 10px 20px;
   cursor: pointer;
   font-size: 1rem;
   color: white;
   font-weight: 700;
-
-  &:hover {
-    background-color: var(--primary-color-shade);
-  }
-
-  ${(props) =>
-    props.$variant === "selected" &&
-    css`
-      &:focus {
-        background-color: var(--secondary-color);
-        transition: 0.5s;
-      }
-    `}
 `;
 
-export default function Button({ children, onClick, $variant }) {
+export default function Button({ children, onClick, $variant, isActive }) {
   return (
-    <StyledButton $variant={$variant} onClick={onClick}>
+    <StyledButton $variant={$variant} isActive={isActive} onClick={onClick}>
```

Comment:
props you pass to styled components should be prefixed with a "$". Otherwise they will be passed to the HTML element as attributes and you will get an error in the console, since they are no native attributes

```suggestion
    <StyledButton $variant={$variant} $isActive={isActive} onClick={onClick}>
```

Reply:
Done.